pages/dkp/kommander/1.1/release-notes/index.md

```diff
@@ -19,13 +19,56 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated and cost management, across multiple clusters, whether they are a new Konvoy cluster or existing 3rd party/DIY distribution.
 
-## Supported Versions
```

Comment:
@cmastr @russurquhart1 Do we want to get rid of this though?

Reply:
I think so, kommander will automatically discover new versions that are supported with each konvoy releae. So eventhough these may be accurate when the version of kommander is released, it will be wrong with the next konvoy patch version release.